libs/designer/src/lib/core/actions/bjsworkflow/templates.ts

```diff
@@ -0,0 +1,214 @@
+import {
+  DevLogger,
+  guid,
+  type ILoggerService,
+  InitApiManagementService,
+  InitAppServiceService,
+  InitConnectionParameterEditorService,
+  InitConnectionService,
+  InitFunctionService,
+  InitGatewayService,
+  InitLoggerService,
+  InitOAuthService,
+  InitOperationManifestService,
+  InitTemplateService,
+  InitTenantService,
+  InitUiInteractionsService,
+  InitWorkflowService,
+  LogEntryLevel,
+  LoggerService,
+  type LogicAppsV2,
+  type Template,
+} from '@microsoft/logic-apps-shared';
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import type { RootState } from '../../state/templates/store';
+import type { TemplateServiceOptions } from '../../templates/TemplatesDesignerContext';
+
+export interface WorkflowTemplateData {
+  name?: string;
+  workflowDefinition: LogicAppsV2.WorkflowDefinition | undefined;
+  manifest: Template.Manifest | undefined;
+  workflowName: string | undefined;
+  kind: string | undefined;
+  images?: Record<string, any>;
```

Comment:
Are we storing images here? Wondering why the Record values are `any` instead of `string`

Reply:
I just copied from previous data model, we should change it to string... will do that